test/unit/api_spec.js

```diff
@@ -168,19 +167,20 @@ describe("api", function () {
           loadingTask.destroy().then(done);
         });
     });
-    it("creates pdf doc from non-existent URL", function (done) {
-      const loadingTask = getDocument(
-        buildGetDocumentParams("non-existent.pdf")
-      );
-      loadingTask.promise
-        .then(function (error) {
-          done.fail("shall fail loading");
-        })
-        .catch(function (error) {
-          expect(error instanceof MissingPDFException).toEqual(true);
-          loadingTask.destroy().then(done);
-        });
-    });
+    // TODO: re-enable https://github.com/mozilla/pdf.js/issues/13077
+    // it("creates pdf doc from non-existent URL", function (done) {
```

Comment:
Given that this doesn't seem to fail when the tests are run with Github Actions, can't we keep this enabled *conditionally* instead of just commenting it out?

Something along these lines instead:
```javascript
if (!isNodeJS) {
  pending(...);
}
```